tests/support/server.tcl

```diff
@@ -536,7 +536,7 @@ proc start_server {options {code undefined}} {
     }
 
     # write new configuration to temporary file
-    set config_file [tmpfile redis.conf]
+    set config_file [tmpfile server.conf]
```

Comment:
I think server.conf is a bit too generic since this is user facing. we should probably go with "valkey.conf"

Reply:
Yeah for me - as a user - I would like a non-generic filename. Makes it easier for me to search for "valkey.conf repl-backlog-size" on search engines, etc. I regularly would search "redis.conf" to pull up the old default configuration. Just my two cents

Reply:
Ok, good points. I'm convinced. So valkey.conf?

I feel this PR is too large. We could have one PR for only the config file name.

We need to make decisions for each of them separately. This decision is not even the same decision as for other files, like valkey-cli and valkey-server.

Reply:
Agree with the decision.